tests/test_cache.py

```diff
@@ -0,0 +1,147 @@
+        """
+        add_plugin(
+            self.alias_placeholder,
+            'TextPlugin',
+            language=self.language,
+            body='Content Alias 1234',
+        )
+        # Create alias plugin inside the page placeholder
+        add_plugin(
+            self.page_placeholder,
+            Alias,
+            language=self.language,
+            alias=self.alias,
+        )
+
+        # Get page via http request
+        with self.login_user_context(self.superuser):
+            response = self.client.get(self.page.get_absolute_url(self.language))
+
+        # Check the response contains the content we added to our page
+        self.assertContains(response, 'Content Alias 1234')
+
+    def test_query_plugin_count(self):
+        """
+        Aliases are cached, so we should see a decrease in the number of queries
+        when rendering once the cache is populated.
+        """
+        # Create an alias, a placeholder and a plugin within the placeholder
+        add_plugin(
+            self.alias_placeholder,
+            'TextPlugin',
+            language=self.language,
+            body='test 2',
+        )
+        add_plugin(
+            self.page_placeholder,
+            'TextPlugin',
+            language=self.language,
+            body='Content Alias 1234',
+        )
+        add_plugin(
+            self.page_placeholder,
+            Alias,
+            language=self.language,
+            alias=self.alias,
+        )
+
+        with self.login_user_context(self.superuser):
+            # render the page
+            self.client.get(self.page.get_absolute_url(self.language))
+        # Get the plugins from the page and count them
+        plugins = self.page_placeholder.get_plugins()
+        plugins_count = plugins.count()
+
+        # Ensure only the two created plugins are present
+        self.assertEqual(plugins_count, 2)
+
+    def test_create_alias_with_default_render_template(self):
+        # Create an alias plugin without explicitly defining the template to use
+        add_plugin(
+            self.alias_placeholder,
+            Alias,
+            language=self.language,
+            alias=self.alias,
+        )
+
+        # The default template should be used if one is not provided
+        self.assertEqual(self.alias.cms_plugins.first().template, 'default')
+
+    def test_create_alias_with_custom_render_template(self):
+        # Define a custom template
+        alias_template = 'custom_alias_template'
+
+        # Create an alias plugin providing our custom template
+        add_plugin(
+            self.alias_placeholder,
+            Alias,
+            language=self.language,
+            alias=self.alias,
+            template=alias_template,
+        )
+
+        # The custom template should be assigned to the plugin we created
+        self.assertEqual(self.alias.cms_plugins.first().template, "custom_alias_template")
+
+    def test_query_count(self):
```

Comment:
This is testing the placeholder cache, we should renamed it to better describe what we are testing:

```suggestion
    def test_placeholder_cache(self):
        """
        The placeholder contents should be cached for published content (or all content if versioning is not installed).
        """
```

Reply:
Renamed the test case